client/network/src/error.rs

```diff
@@ -61,6 +61,8 @@ pub enum Error {
 		/// The invalid addresses.
 		addresses: Vec<Multiaddr>,
 	},
+	/// Unable to fetch storage error.
+	InvalidStorage,
```

Comment:
This enum contains errors that can happen at initialization. The storage value can become invalid at any future or past block. Therefore it doesn't make sense to me to add this.